packages/storybook/stories/Troubleshooting.stories.mdx

```diff
@@ -98,3 +98,32 @@ can be caused by passing arguments to our factory functions too often. We recomm
 or disposing of the adapter or stateful client before attempting to create a new one.
 
 Another common cause for this error is using strict mode in your application. Please see [React's documentation](https://react.dev/reference/react/StrictMode) on strict mode to learn more.
+
+## Mobile browser refresh
+
+This is an issue when using a mobile browser and the user closes the browser. When they do this
+the browser will load the page cached which all of the subscriptions will be lost. To fix this as the
```

Comment:
For the second sentence, do you mean? "When they do this
the browser will load the page but all of the cached subscriptions will be lost"